nservicebus/sqlserver/transactions_native_sqlserver_[,3).partial.md

```diff
@@ -0,0 +1,8 @@
+There are two available options within native transaction level:
```

Comment:
seems those two partials should go the other way round? this one is the [3,x) right?